src/main/java/org/mule/extension/ftp/internal/FtpOperations.java

```diff
@@ -156,7 +161,15 @@ public void write(@Config FileConnectorConfig config, @Connection FileSystem fil
                     @Optional(defaultValue = "true") boolean createParentDirectories,
                     @Optional(defaultValue = "false") boolean lock, @Optional(
                         defaultValue = "OVERWRITE") @Summary("How the file is going to be written") @DisplayName("Write Mode") FileWriteMode mode) {
-    super.doWrite(config, fileSystem, path, content, createParentDirectories, lock, mode);
+    if (content == null) {
+      throw new IllegalContentException("Cannot write a null content");
+    }
+
+    if (isBlank(path)) {
+      throw new IllegalPathException(format("path cannot be null nor blank"));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: String contains no format specifiers. ([squid:S3457](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3457))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-ftp-connector&pullRequest=103&issues=AWmW4ZlJotIXnmsU8B5V&open=AWmW4ZlJotIXnmsU8B5V)

[ISSUE_KEY]: AWmW4ZlJotIXnmsU8B5V